frontend/src/metabase/query_builder/components/DataSelector/saved-question-picker/SavedQuestionPicker.jsx

```diff
@@ -150,7 +150,7 @@ export default _.compose(
     loadingAndErrorWrapper: false,
   }),
   Collection.loadList({
-    query: () => ({ tree: true }),
+    query: () => ({ tree: true, "exclude-archived": true }),
```

Comment:
Does this trigger a full load every time? Wondering because sometimes I don't see network activity when I click the saved question picker. We do need to ensure that this component has the full question set since we load the reduced set on load.

Reply:
Not every time, only once during the lifetime of the browser tab